src/internal/stream/doc.go

```diff
@@ -0,0 +1,2 @@
+// package stream contains interfaces and helper functions for managing iterators that can block.
```

Comment:
package -> Package.  They made the convention "Package x ..." so that the sentence can start with an uppercase letter instead of the lowercase letter starting the package name :D